app/views/comments/_form.html.erb

```diff
@@ -1,3 +1,4 @@
+<div style="background-color:#DCDCDC;border: 1px solid #bbb;border-bottom: 0;padding: 10px;">
```

Comment:
```suggestion
<div style="background-color:#f8f8f8; border: 1px solid #e7e7e7;padding: 18px;">
```

Reply:
OK I do like that design better, thanks!